RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -0,0 +1,65 @@
+import FWCore.ParameterSet.Config as cms
+
+calibratedEgammaSettings = cms.PSet(minEtToCalibrate = cms.double(5.0),
+                                    semiDeterministic = cms.bool(True),
+                                    correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_17Nov2017_v1_ele_unc"),
+                                    recHitCollectionEB = cms.InputTag('reducedEcalRecHitsEB'),
+                                    recHitCollectionEE = cms.InputTag('reducedEcalRecHitsEE'),
+                                    produceCalibratedObjs = cms.bool(True)
+                                    )
+calibratedEgammaPatSettings = calibratedEgammaSettings.clone(
+    recHitCollectionEB = cms.InputTag('reducedEcalRecHitsEB'),
+    recHitCollectionEE = cms.InputTag('reducedEcalRecHitsEE'),
+    )
+
+ecalTrkCombinationRegression = cms.PSet(
+    ecalTrkRegressionConfig = cms.PSet(
```

Comment:
this is already generated (hopefully appropriately) inside calibratedElectronProducer_cfi.
Why is the extra copy needed?